### PR TITLE
Update SECURITY.md

### PR DESCRIPTION
**Pull request by mikkeloscar**
Update to match example: https://github.com/zalando-incubator/new-project/blob/master/SECURITY.md

### CODE REVIEW

**Comment by mikkeloscar** [comment]
:+1:

File: SECURITY.md [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
-channels:
+We host our bug bounty program on HackerOne, it is currently private, therefore if you would like to report a vulnerability and get rewarded for it, please ask to join our program by filling this form:
 
-Email: Tech-Security@zalando.de
-OR
-Submit your vulnerability report through our bug bounty program at:
-https://hackerone.com/zalando
+https://corporate.zalando.com/en/services-and-contact#security-form
+
+You can also send you report via this form if you do not want to join our bug bounty program and just want to report a vulnerability or security issue.
```
Comment (hjacobs) [inline SECURITY.md]: typo: "you" -> "your" (maybe the template is wrong?)

Comment (mikkeloscar) [reply SECURITY.md]: yes template is wrong :'(

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**